NebulaNetwork/Messaging/WebSocketService.cs

```diff
@@ -81,19 +85,22 @@ protected override void OnClose(CloseEventArgs e)
 
     protected override void OnError(ErrorEventArgs e)
     {
+        if (!connections.TryGetValue(Context.UserEndPoint.GetHashCode(), out var connection))
+        {
+            return;
+        }
```

Comment:
Why is this needed?

According to https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.remove?view=netstandard-2.0, remove() returns true if the element is found and removed or false if it is not found, so in this case the TryGetValue should just be redundant and result in an additional lookup.